src/qt/sendcoinsdialog.cpp

```diff
@@ -305,9 +313,19 @@ void SendCoinsDialog::on_sendButton_clicked()
         formatted.append(recipientElement);
     }
 
-    QString questionString = tr("Are you sure you want to send?");
+    QString questionString;
+    if (!model->privateKeysDisabled()) {
+        questionString.append(tr("Are you sure you want to send?"));
+    } else {
+        questionString.append(tr("Do you want to draft this transaction?"));
+    }
+
     questionString.append("<br /><span style='font-size:10pt;'>");
-    questionString.append(tr("Please, review your transaction."));
+    if (!model->privateKeysDisabled()) {
+        questionString.append(tr("Please, review your transaction."));
+    } else {
+        questionString.append(tr("Please, review your transaction proposal. This will produce a Partially Signed Bitcoin Transaction (PSBT) which you can copy and then sign with e.g. an offline Bitcoin Core wallet, or a PSBT-compatible hardware wallet."));
```

Comment:
Not sure we need to change this message.

Reply:
This is the only place where it's not in a toolbox.

Reply:
???

Reply:
I mean tool tip.